flaskProject/add_data.py

```diff
@@ -20,7 +20,7 @@ def init_db():
         date_estreno="28/10/2020",
         model_generic="basic",
         last_coordinate_latitude=23.4433,
-        last_coordinate_longitude=23.4433,
+        last_coordinate_longitude=23.4432,
```

Comment:
I think that there are not multiples coord for a single moto, so the name of the attribute may be shortened with leaving out"last", or simply coord_lon and coord_lat